docs/getting_started.md

```diff
@@ -0,0 +1,97 @@
+# Getting Started
+
+Once you have [installed the operator](./installation.md) you can follow this guide to deploy an example application demonstrating the operator.
+
+### Deploy a `FeatureFlagConfiguration`
+
+This `FeatureFlagConfiguration` will be watched by the injected `flagd` container and used to construct its internal flag definitions state. If multiple configurations are supplied to `flagd` these states will be merged.
```

Comment:
```suggestion
This `FeatureFlagConfiguration` is watched by the injected `flagd` container and used to construct its internal flag definitions state. If multiple configurations are supplied to `flagd` these states will be merged.
```

I generall recommend using present tense,  imperative mood and passive voice for documentation.

So avoid "will be" and just say "is". This sentence is already passive voice, so that's fine, but avoid the tense shifts.